### DISCUSSION TITLE
Less of an idea, more of a request: Option to make charging icon static and not flashing in menu bar

### DISCUSSION THREAD

**Discussion by adam-edword** [thebarbican19/BatteryBoi#53] [Ideas]
Title really, would be nice to have the option to leave the little lightning bolt always on, instead of it flashing like it currently does!

**Comment by adam-edword** [comment]
Wasn't sure where to put this since there isn't a feedback/requests channel :[

**Reply by thebarbican19** [reply]
Love it. Yeah working on ideas to make this more customizable.

**Reply by adam-edword** [reply]
Heroic
